### PR TITLE
Black formatting

### PR DESCRIPTION
**Pull request by a-ws-m**
Use `black` to format two files, per [this request](https://github.com/Becksteinlab/MDPOW/pull/267#pullrequestreview-1585104049).

### CODE REVIEW

**Comment by orbeckst** [comment]
Let's just switch _everything_ over to black #271 — I wanted to try that out in a project anyway.

@a-ws-m could you do the honors and blacken the whole code base, please?

**Review by orbeckst — CHANGES_REQUESTED** [review]
See #271 — please 
- blacken *all code* (in `mdpow`, `mdpow/tests`, and `scripts`)
- add yourself to AUTHORS
- add an entry to CHANGES

**Comment by a-ws-m** [comment]
Done, I also:

* Added a `.git-blame-ignore-revs` per [the suggestion on that link you sent](https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html#avoiding-ruining-git-blame).
* Added some files in `.vscode` so that users of that editor will be recommended to install `black` and have it format Python files upon saving.

**Review by orbeckst — APPROVED** [review]
Thank you! Looking good.

There are minor formatting changes for the CHANGES. You can just accept my "suggestions" from the comments.

My original plan was to squash-merge. However, if you're familiar with `git` then you could rewrite the history of this branch into **two commits**:
1. **all** black changes 
2. all other changes (AUTHORS, CHANGES, workflows, updated .git-blame-ignore-revs with the commit ID of (1))

Then I will be able to just merge without any need for any further changes.

Let me know if you want to rewrite the history. Otherwise I'll squash-merge after the code review suggestions are included and tests pass.

File: .git-blame-ignore-revs
Comment (orbeckst) [reply .git-blame-ignore-revs:1]: (Unless you rewrite the history yourself and add the proper commit ID of the single commit with all black-generated changes. See review comment.)

Comment (a-ws-m) [reply .git-blame-ignore-revs:1]: Hi, I'm not confident I'll get that right to be honest with you! If it's OK, I'll leave it to you to change the hashes after merging.

Comment (orbeckst) [reply .git-blame-ignore-revs:1]: ok, will do

**Comment by orbeckst** [comment]
I rewrote the history.

**Comment by orbeckst** [comment]
Alright, we're now using `black`!

**Pull request merged**
